docs/quick-starts/03-quick-start-cloud-run.md

```diff
@@ -0,0 +1,120 @@
+# Quick Start: Cloud Run
+
+This quick start discusses how to build and deploy a function using
+[Cloud Run], a serverless platform on [Google Cloud].
+
+The instructions in this quick start can be run from a terminal on your own
+machine or from [Cloud Shell], an online development and operations environment,
+using the [Cloud Shell Editor].
+
+Using the [Cloud Shell Editor] is beyond the scope of this quick start, but
+this [codelab] can help you get started.
```

Comment:
`code lab` – two words?

Reply:
Leave it. It's officially "Google Codelabs"

Reply:
https://codelabs.developers.google.com/